dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/services/BulkDataRequestManager.kt

```diff
@@ -77,21 +77,7 @@ class BulkDataRequestManager(
         val currentUserId = DatalandAuthentication.fromContext().userId
         val retrievedStoredDataRequestEntitiesForUser = dataRequestRepository.findByUserId(currentUserId)
         val retrievedStoredDataRequestsForUser = retrievedStoredDataRequestEntitiesForUser.map { dataRequestEntity ->
-            StoredDataRequest(
-                dataRequestEntity.dataRequestId,
-                dataRequestEntity.userId,
-                dataRequestEntity.creationTimestamp,
-                utils.getDataTypeEnumForFrameworkName(dataRequestEntity.dataTypeName),
-                dataRequestEntity.reportingPeriod,
-                dataRequestEntity.dataRequestCompanyIdentifierType,
-                dataRequestEntity.dataRequestCompanyIdentifierValue,
-                objectMapper.readValue(
```

Comment:
Wieso ist die messageHistory eigentlich nicht einfach nullable?
Würde man sich dadurch nicht eine Menge Ärger ersparen, weil man dann nicht immer leere Listen da reinpacken müsste?

Und warum ist "dataType" eigentlich nullable in "StoredDataRequest" ?
Sollte nicht jeder data request einen eindeutigen dataType-Wert haben?
Immerhin ist die Kombination dataType-reportingPeriod-companyId-userId der primary key zu jeder einzelnen stored datarequest oder?